README.md

```diff
@@ -80,77 +80,82 @@ The platform is built on top of [Ambire Wallet](https://www.ambire.com) and uses
+Layer 2 refers to blockchain scaling solutions, which allow financial transactions or other state transitions to happen very fast, off the blockchain, while still being enforceable or eventually being committed to the underlying blockchain. Ideally, Layer 2 solutions allow throughput levels seen in centralized systems, while still being as trustless and censorship-resistant as blockchains. In AdEx, we use a scaling primitive, called OUTPACE, which allows for multiple campaigns to run on the same payment channel, thus allowing for lower transaction fees.
 
-In the AdEx protocol, each campaign uses a payment channel powered by a technology we call **OUTPACE**. Multiple campaigns can run on the same payment channel, as payment channels are created between **validators** (more on this later).
+#### Validators
+Validators in the AdEx ecosystem are responsible for verifying and validating transactions on the blockchain, ensuring the security and transparency of the network. More about validators can be found here. 
 
-#### Layer 2
+#### Observers
+The observers are delegated to collect events in relation to a certain campaign. All validators of a campaign are, by definition, observers of all events related to it.
 
-Layer 2 refers to blockchain scaling solutions, which allow financial transactions or other state transitions to happen very fast, off the blockchain, while still being enforceable or eventually being committed to the underlying blockchain.
 
-Ideally, layer 2 solutions allow throughput levels seen in centralized systems, while still being as trustless and censorship-resistant as blockchains.
+### Layer 2
 
-In AdEx, we use two scaling primitives that we defined: **OCEAN** and **OUTPACE**.
+#### OUTPACE
 
-#### Offchain unidirectional trust-less payment channel (OUTPACE)
+In the AdEx protocol, each campaign uses a payment channel powered by a technology we call OUTPACE. Multiple campaigns can run on the same payment channel, as payment channels are created between validators.
 
 **OUTPACE** stands for **O**ffchain **u**nidirectional **t**rustless **pa**yment **c**hann**e**l
 
 An **OUTPACE** channel is defined on-chain with a validator set and a token address. Any amount of funds of that token can be deposited into the channel at any time. The validators observe off-chain events, and the leading validator (validators[0]) would propose the new state of the channel, and the rest of the validators check and confirm those new states.
 
-If a state is signed by a supermajority (>=2/3) of validators, it can be used to enforce a result on-chain and withdraw funds from the channel.
+If a state is signed by a supermajority (>=2/2) of validators, in the case of campaigns in which direct publishers participate, it can be used to enforce a result on-chain and withdraw funds from the channel.
```

Comment:
technically it cannot be >2/2 so change >=2/2 to 2/2

Reply:
also, SSPs still participate in the channel, but we are controlling the wallet to which the money goes in order to be able to take those funds and pay them FIAT. So the "direct publishers" clarification might be redundant